### PR TITLE
LPD-38800 | Test Fix | Poshi >>> Playwright

### PR DESCRIPTION
**Pull request by rafaellajordao**
### Motivation:
Test failing due to Poshi limitations.

### Proposed Solution:
Transcribe test to playwright in order to facilitate maintenance.

**More details:** [LPD-38800](https://liferay.atlassian.net/browse/LPD-38800)

### CODE REVIEW

**Comment by rafaellajordao** [comment]
ci:test:sf

**Comment by rafaellajordao** [comment]
ci:test:sf

**Comment by rafaellajordao** [comment]
ci:test:analytics-cloud

File: modules/test/playwright/tests/osb-faro-web/eventAnalysis.spec.ts [hunk 1]
```diff
@@ -323,6 +324,541 @@ temp: "11",
...
+		await test.step('Go to Analytics Cloud and Switch the property', async () => {
+			await navigateToACPageViaURL({
+				acPage: ACPage.eventAnalysisPage,
+				channelID: channel.id,
+				page,
+				projectID: project.groupId,
+			});
+		});
+
+		await test.step('Add the custom event, create an analysis and add a brekdown', async () => {
```
Comment (interaminense) [inline modules/test/playwright/tests/osb-faro-web/eventAnalysis.spec.ts:644]: Rename step description

File: modules/test/playwright/tests/osb-faro-web/eventAnalysis.spec.ts [hunk 2]
```diff
@@ -323,6 +324,541 @@ temp: "11",
...
+			await page.getByRole('link', {name: 'Create Analysis'}).click();
+		});
+
+		await test.step('Add a name to the segment', async () => {
```
Comment (interaminense) [inline modules/test/playwright/tests/osb-faro-web/eventAnalysis.spec.ts:648]: `Add a name to the segment` -> `Add a name to the event analysis`

File: modules/test/playwright/tests/osb-faro-web/eventAnalysis.spec.ts [hunk 3]
```diff
@@ -323,6 +324,541 @@ temp: "11",
...
+		await test.step('Go to Analytics Cloud and Switch the property', async () => {
+			await navigateToACPageViaURL({
+				acPage: ACPage.eventAnalysisPage,
+				channelID: channel.id,
+				page,
+				projectID: project.groupId,
+			});
+		});
+
+		await test.step('Add the custom event, create an analysis and add a brekdown', async () => {
```
Comment (interaminense) [inline modules/test/playwright/tests/osb-faro-web/eventAnalysis.spec.ts:776]: Rename this step description as well?

File: modules/test/playwright/tests/osb-faro-web/eventAnalysis.spec.ts [hunk 4]
```diff
@@ -323,6 +324,541 @@ temp: "11",
...
+			await page.getByRole('link', {name: 'Create Analysis'}).click();
+		});
+
+		await test.step('Add a name to the segment', async () => {
```
Comment (interaminense) [inline modules/test/playwright/tests/osb-faro-web/eventAnalysis.spec.ts:780]: `Add a name to the segment` -> `Add a name to the analysis`

**Comment by rafaellajordao** [comment]
ci:test:sf

**Comment by interaminense** [comment]
ci:forward

**Pull request closed without merging**